daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/Value.scala

```diff
@@ -499,6 +517,13 @@ private final class `Value Order instance`[Cid: Order](Scope: Value.LookupVarian
         },
       )
     case ValueRecord(_, a) => (140, k { case ValueRecord(_, b) => _2.T.subst(a) ?|? _2.T.subst(b) })
+    case ValueBuiltinException(tagA, a) =>
+      (
+        150,
+        k { case ValueBuiltinException(tagB, b) =>
+          tagA ?|? tagB |+| a ?|? b
+        },
+      )
```

Comment:
this problably should be tested in ValueSpec.

Reply:
Added a generator for BuiltinException, but it is not enabled because it causes lots of test fails.
Lets talk about this tomorrow!...